### PR TITLE
Added error handling for @angular/language-service calls

### LINKED ISSUE #128: Output Window gets opened when I type
Output window gets opened any time I type some code and shows language service output.

When I close it doesn't have permanent effect, it gets opened again when I type.

I can't see any settings in the plugin. What shall I do? I don't want Output window open.

### PR DESCRIPTION
**Pull request by DBosley**
This PR should fix #128 for any errors thrown by the following @angular/language-service calls:
```
getHoverAt
getDefinitionAt
getTemplateReferences
getDiagnostics
getCompletionsAt
```

### CODE REVIEW

File: server/src/editorServices.ts [hunk 1]
```diff
@@ -3035,7 +3035,15 @@ export class LineLeaf implements LineCollection {
...
   function time<T>(name: string, cb: () => T): T {
     const start = Date.now();
-    const result = cb();
+    let result: T = null;
+    try{
+        result = cb();
+    } catch (error) {
+        logger.msg(
+            `Error for ${name}:\n` +
+            `    ${error}`
```
Comment (chuckjaz) [inline server/src/editorServices.ts]: The stack is useful to diagnose problems. Consider changing this to `error.stack || error`

**Comment by chuckjaz** [comment]
Let me know if you want to take my recommendation. I will commit this before the release as well.

**Comment by DBosley** [comment]
@chuckjaz I'll toss that in there. One minute.

**Comment by DBosley** [comment]
@chuckjaz All set!

**Pull request merged**